public/js/graph/react_node.js.jsx

```diff
@@ -50,26 +50,61 @@ function getNodes(mode){
         entry['style'] = getStyles(entry['attributes']['style']);
         entry['children'] = [];
         //value.children is an HTML collection, converting to array here
-        var children_arr = Array.prototype.slice.call(element.children);
+        var childrenArr = Array.prototype.slice.call(element.children);
         //Assumed only one level of children, the <rect> and one or more <text>
-        children_arr.forEach(function(child) {
-            var child_entry = {};
-            child_entry['attributes'] = getAttributes(child.attributes);
-            child_entry['style'] = getStyles(child_entry['attributes']['style']);
+        childrenArr.forEach(function(child) {
+            var childEntry = {};
+            childEntry['attributes'] = getAttributes(child.attributes);
+            childEntry['style'] = getStyles(childEntry['attributes']['style']);
             //innerHTML is just for text within the <text>
             //there aren't anymore children since it was assumed only one level of children
-            child_entry['innerHTML'] = child.innerHTML;
-            entry['children'].push(child_entry);
+            childEntry['innerHTML'] = child.innerHTML;
+            entry['children'].push(childEntry);
         });
-        dict_list.push(entry);
+        dictList.push(entry);
     });
-    return dict_list;
+    return dictList;
 }
 
+//Quick fix to initiallyTakeable case-sensitivity
+//Will later remove initiallyTakeable, by making a check if prerequisite list is empty or not
+var initiallyTakeableUpper = $.map(initiallyTakeable, function(item, index) {
+    return item.toUpperCase();
+});
+
 var ReactSVG = React.createClass({
+    componentDidMount: function(){
+        //Need to hardcode these in because React does not understand these attributes
+        var svgNode = React.findDOMNode(this.refs.svg);
+        var markerNode = React.findDOMNode(this.refs.marker);
+        
+        svgNode.setAttribute('xmlns','http://www.w3.org/2000/svg');
+        svgNode.setAttribute('xmlns:xlink','http://www.w3.org/1999/xlink');
+        svgNode.setAttribute('xmlns:svg','http://www.w3.org/2000/svg');
+        svgNode.setAttribute('xmlns:dc','http://purl.org/dc/elements/1.1/');
+        svgNode.setAttribute('xmlns:cc','http://creativecommons.org/ns#');
+        svgNode.setAttribute('xmlns:rdf','http://www.w3.org/1999/02/22-rdf-syntax-ns#');
+        
+        markerNode.setAttribute('viewBox', '0 0 10 10');
```

Comment:
I see a couple of these attributes [here](https://facebook.github.io/react/docs/tags-and-attributes.html). Is this a version issue?